LLVM_AST/llvm_ast.sml

```diff
@@ -0,0 +1,97 @@
+(*********************************************************************************)
```

Comment:
Rename the files.

1. Create a directory `llvm` and inside that have a file `ast.sml`

2. Also create a file llvm.sml in the root directory

3. Inside ast.sml create structure called ast

4. Inside the llvm directory  have a module LLVM and inside that structure Ast = ast.